src/pages/signing_official_console/DAACell.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Checkbox } from '@mui/material';
+import { DAA } from '../../libs/ajax/DAA';
+import { Notifications } from '../../libs/utils';
+
+export default function DAACell(rowDac, researcher, institutionId, daas, refreshResearchers, setResearchers) {
+  const id = researcher && (researcher.userId || researcher.email);
+  const libraryCards = researcher && researcher.libraryCards;
+  const card = libraryCards && libraryCards.find(card => card.institutionId === institutionId);
+  const daaIds = researcher && card && card.daaIds;
+  const filteredDaas = daaIds && daas.filter(daa => daaIds.includes(daa.daaId));
+  const hasDacId = filteredDaas && filteredDaas.some(daa => daa.dacs.some(dac => dac.dacId === rowDac.dacId));
+
+  const handleClick = async (researcher, specificDac, filteredDaas, checked, refreshResearchers, setResearchers) => {
```

Comment:
Similar to previous comment, if all of these were destructured from `props` then you wouldn't need such a large number of arguments here ... they would all be predefined in this instance of the component.